packages/amplication-git-pull-request-service/src/core/commit/dto/pull-request-comment-details.dto.ts

```diff
@@ -0,0 +1,3 @@
+export interface PullRequestCommentDetails {
+  url: string;
+}
```

Comment:
DTOs should be implemented as classes in NestJS ([docs](https://docs.nestjs.com/techniques/validation)).

Also, please ensure you naming follows NestJS best practices and the [practices we presented last month](https://www.notion.so/amplication/File-Declaration-Naming-666afe407ff14ed6aa54bcd3dead45a8).

More specifically, you need to add the `Dto` ending to the name of the DTO.